platform/integrations/rancher/configuration.mdx

```diff
@@ -74,64 +84,26 @@ Now we'll add the Rancher object to the [Platform configuration](../../api/resou
 
   <TabItem value="token_inline">
 
-```yaml
-auth:
-  rancher:
-    host: your-rancher-instance
-    bearerToken: "your-rancher-token"
-    insecure: <true/false>
-```
+    ```yaml
+    auth:
+      rancher:
+        host: <RANCHER_SERVER_URL>
```

Comment:
nit: Should we call this Rancher server Hostname instead, because we do ask users to 

> Replace `<RANCHER_SERVER_URL>` with the address of your Rancher instance without the protocol prefix, for example, my-rancher.demo.dev

Reply:
Yes, I blanked out on what to call it and new URL wasn't right but never came back to it. Good catch.